src/com/goide/runconfig/testing/GoTestRunningState.java

```diff
@@ -83,7 +83,10 @@ protected GoExecutor patchExecutor(@NotNull GoExecutor executor) throws Executio
         String relativePath = FileUtil.getRelativePath(myConfiguration.getWorkingDirectory(),
                                                        myConfiguration.getDirectoryPath(),
                                                        File.separatorChar);
-        if (relativePath != null) {
+        if (relativePath == "." && myCoverageFilePath != null) {
```

Comment:
write a todo to remove this as soon as multiple packages coverage will be supported with the link to corresponding issue